CIP-015x/README.md

```diff
@@ -169,13 +169,14 @@ Updating of the file in-place is unsafe so instead we store list of updated.
 All updates are written in the following way:
 
 - if an entry has a natural key and we want to update data we can store a new value;
-- if we want to remove a value with natural key we should write a special tombstone entry, see namespaces;
+- if we want to remove a value with natural key we should write a special tombstone entry;
```

Comment:
I think we could do with being more consistent about the extent to which we support non-keyed entries. In the CHUNK section above we suggest that we do require them. But then here we discuss entries without keys, and below we have what looks like support for updating keyless entries.

My feeling is that we should require keys. We need something to define ordering, for single-entry records we can just use a key of 0 (keys need only be unique within a namespace), and for small lists that have a natural order we can just use nat indices. I guess our only concern is structures that are naturally list-structured but where the element is large and has no natural key - but these are going to be a pain for us anyway.

Reply:
Agree to have a requirement for the keys. I'll go through the document and check all the wording to be sure that we require keys explicitly.